### DISCUSSION TITLE
Is it possible to ignore a node?

### DISCUSSION THREAD

**Discussion by togakangaroo** [org-roam/org-roam-ui#278] [Q&A] [answered]
I have a note called "Daily Notes" where I enter a running log of all my notes don't fit better into specific areas. I cross link from there liberally. As you can imagine, this ends up being linked to nearly every other node. Is it possible to tell org-roam-ui to ignore this one roam node?

The project is great, thank you so much!

**Comment by tefkah** [comment, marked as answer]
you can one time hide it from the graph by right-clicking it, or you can give it a tag and add it to the blocklist!
